src/valkey-benchmark.c

```diff
@@ -433,6 +464,32 @@ static void setClusterKeyHashTag(client c) {
     }
 }
 
+static long long acquireTokenOrWait(int tokens) {
+    if (config.num_threads) pthread_mutex_lock(&(config.token_bucket_mutex));
+
+    long long last_time_ = config.last_time;
+    long long time_per_token_ = config.time_per_token;
+    long long time_per_burst_ = config.time_per_burst;
```

Comment:
Trailing underscore in variable names is a funny style. We don't have this style in this project so far. I prefer that we don't start using it.

```suggestion
    long long last_time = config.last_time;
    long long time_per_token = config.time_per_token;
    long long time_per_burst = config.time_per_burst;
```